frontend/.eslintrc.json

```diff
@@ -49,7 +49,7 @@
         "reservedFirst": true
       }
     ],
-    "react/jsx-filename-extension": [1, { "extensions": [".js"] }],
+    "react/jsx-filename-extension": [1, { "extensions": [".jsx", ".tsx"] }],
```

Comment:
Is the .jsx rename necessary?

Reply:
Using JSX in `.js` files complicated vitest setup. See https://github.com/vitejs/vite/discussions/3448